params/config.go

```diff
@@ -1140,20 +1140,21 @@ var (
 	testnetBytePrefixList = make([][]int, 20000)
 )
 
+// Guarded Address Space ranges
 func init() {
-	mainnetBytePrefixList[9000] = []int{0, 9}
-	mainnetBytePrefixList[9100] = []int{10, 19}
-	mainnetBytePrefixList[9101] = []int{20, 29}
-	mainnetBytePrefixList[9102] = []int{30, 39}
-	mainnetBytePrefixList[9103] = []int{40, 49}
-	mainnetBytePrefixList[9200] = []int{50, 59}
-	mainnetBytePrefixList[9201] = []int{60, 69}
-	mainnetBytePrefixList[9202] = []int{70, 79}
-	mainnetBytePrefixList[9203] = []int{80, 89}
-	mainnetBytePrefixList[9300] = []int{90, 99}
-	mainnetBytePrefixList[9301] = []int{100, 109}
-	mainnetBytePrefixList[9302] = []int{110, 119}
-	mainnetBytePrefixList[9303] = []int{120, 129}
+	mainnetBytePrefixList[9000] = []int{0, 9}     // Prime
+	mainnetBytePrefixList[9100] = []int{10, 19}   // Cyprus
+	mainnetBytePrefixList[9101] = []int{20, 29}   // Cyprus One
+	mainnetBytePrefixList[9102] = []int{30, 39}   // Cyprus Two
+	mainnetBytePrefixList[9103] = []int{40, 49}   // Cyprus Three
+	mainnetBytePrefixList[9200] = []int{50, 59}   // Paxos
+	mainnetBytePrefixList[9201] = []int{60, 69}   // Paxos One
+	mainnetBytePrefixList[9202] = []int{70, 79}   // Paxos Two
```

Comment:
@BryceAgent comments are nice here but until we solidify the PR lets hold off for clarity.

Reply:
Agreed, this PR should only feature changes to chain_makers.go and chain_makers_test.go. Other changes should be reverted to main.